### PR TITLE
helath-checkpoint endpoint

### PR DESCRIPTION
**Pull request by basicem**

### CODE REVIEW

**Comment by amarbasic** [comment]
Changes:

1. startup.js

instead of  `require('./app/routes/health-check.routes')(app);` use `app.use(require('./app/routes'));`

2. create `routes/index.js` file and add

```
var router = require('express').Router();

router.use('/api/health-check', require('./health-check.routes'));

module.exports = router;
```

3. Change `health-check.routes.js`:

```
const router = require('express').Router();
const healthCheck = require('../controllers/health-check.controller.js');

router.get('/', healthCheck.status);

module.exports = router;

```

File: startup.js [hunk 1]
```diff
@@ -1,10 +1,16 @@
...
+app.use(bodyParser.json());
 
 app.get('/', (req, res) => {
-  res.send('Hello :)');
+  res.json({ status: 'Prvi prolaz :)' });
 });
 
+app.use(require('./app/routes'));
+
+// require('./app/routes/health-check.routes')(app);
```
Comment (amarbasic) [inline startup.js]: Remove commented code

File: startup.js [hunk 2]
```diff
@@ -1,10 +1,16 @@
+const bodyParser = require('body-parser');
 const express = require('express');
 
 const app = express();
 const port = process.env.PORT;
+app.use(bodyParser.json());
 
 app.get('/', (req, res) => {
```
Comment (amarbasic) [inline startup.js:6]: you can remove this endpoint

File: startup.js [hunk 3]
```diff
@@ -1,10 +1,16 @@
+const bodyParser = require('body-parser');
 const express = require('express');
 
 const app = express();
 const port = process.env.PORT;
```
Comment (amarbasic) [inline startup.js:4]: please add space before `app.user`

**Review by amarbasic — APPROVED** [review]
looks good.
after fixing few small comments you can merge

File: startup.js (hunk 1 above)
Comment (basicem) [reply startup.js]: ok :(

**Pull request merged**